proxyfsd/logging.conf

```diff
@@ -4,7 +4,8 @@ LogFilePath: proxyfsd.log
 # NOTE: Log levels other than Trace and Debug are always on.
 
 # Enable trace logging on a per-package basis. Trace logs are disabled by default unless enabled here.
-# Supported values: jrpcfs, inode, none (default).
+# Supported values: any combination of "dlm", "fs", "fuse", "headhunter", "inode", "jrpcfs", "logger",
+# and "swiftclient" or "none" (default).
```

Comment:
Same comment as above... the list of packages may have grown since this was put together...